### DISCUSSION TITLE
deploying with docker in Intranet，can not connect to the Internet，all project does not report vulnerabilities

### DISCUSSION THREAD

**Discussion by wuruyu181** [DependencyTrack/dependency-track#1954] [Q&A]
deploying with docker in Intranet，
Intranet can not connect to the Internet，when i upload SBOM，get 0 vulnerabilities：
![橙讯图片_1662688706881](https://user-images.githubusercontent.com/113153957/189256551-d2ca3c7d-a37c-49bc-939a-0a2437f21e90.jpeg)
here is the logs：
![橙讯图片_1662688821975](https://user-images.githubusercontent.com/113153957/189256611-991440d2-356f-4bcf-9a49-c11806f9a24f.jpeg)
![橙讯图片_1662688832675](https://user-images.githubusercontent.com/113153957/189256623-1d8029b1-4c8d-4e78-adf7-2b31a38253bb.jpeg)
my question is:denpendency-track can not depolying in private Intranet？

**Comment by valentijnscholten** [comment]
Dependency Track needs to access the internet to download vulnerability databases and version info from package repositories. See also https://docs.dependencytrack.org/integrations/ecosystem/ 
You may have to configure a proxy to be able to access the internet depending on your companies network: https://github.com/DependencyTrack/dependency-track/blob/e9304da3beba4776784da9104edcefbf6da0b32f/src/main/docker/docker-compose.yml#L59-L64

**Comment by wuruyu181** [comment]
> Dependency Track needs to access the internet to download vulnerability databases and version info from package repositories. See also https://docs.dependencytrack.org/integrations/ecosystem/ You may have to configure a proxy to be able to access the internet depending on your companies network:
> 
> https://github.com/DependencyTrack/dependency-track/blob/e9304da3beba4776784da9104edcefbf6da0b32f/src/main/docker/docker-compose.yml#L59-L64

we have  download vulnerability databases in internet，and deploying the denpendency-track and databases in  private Intranet，we can san nist and vulnerabilities files。
my questions is：denpendency-track can analyzeing in local  vulnerability databases？

**Comment by wuruyu181** [comment]
> Dependency Track needs to access the internet to download vulnerability databases and version info from package repositories. See also https://docs.dependencytrack.org/integrations/ecosystem/ You may have to configure a proxy to be able to access the internet depending on your companies network:
> 
> https://github.com/DependencyTrack/dependency-track/blob/e9304da3beba4776784da9104edcefbf6da0b32f/src/main/docker/docker-compose.yml#L59-L64

I installed versions is v4.5.0，my question is：can denpendency-track integrated with denpendency-check and how？
or can i use external vulnerability databases ？
